aiida/backends/sqlalchemy/tests/nodes.py

```diff
@@ -22,8 +22,8 @@ class TestTransitiveClosureDeletionSQLA(AiidaTestCase):
     """
     def test_creation_and_deletion(self):
```

Comment:
Same comment here - now these tests and the one of Django should be identical?

Reply:
same answer here